src/test/java/ch/puzzle/eft/controller/ExamDownloadControllerTest.java

```diff
@@ -0,0 +1,65 @@
+package ch.puzzle.eft.controller;
+
+import ch.puzzle.eft.service.ExamFileService;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.security.test.context.support.WithMockUser;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.ResultActions;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.io.File;
+import java.nio.file.Files;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@WebMvcTest(ExamDownloadController.class)
+@WithMockUser
+public class ExamDownloadControllerTest {
```

Comment:
Remove public modifier. See this commit -> https://github.com/puzzle/unilu-pruefungsabfrage/commit/3cee12f8288096079d22043411b2a194d3c194df